File: 217-contains-duplicate/contains-duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        HashSet<int> volt = new HashSet<int>();
        for(int i = 0; i < nums.Length; i++)
        {
            if(volt.Contains(nums[i]))
            {
                return true;
            }
            else
            {
                volt.Add(nums[i]);
            }
        }
        return false;
    }
}

File: 217-contains-duplicate/contains-duplicate.cpp
auto init = []()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);
    return 0;
}();

class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        std::unordered_set<int> mySet;
        for(int i : nums){
            if(!mySet.insert(i).second){
                return true;
            }
        }
    return false;
    }
};
